src/TiogaBlock.cpp

```diff
@@ -785,7 +785,6 @@ double TiogaBlock::update_solution(const int nvars)
     if (num_nodes_ < 1) return rnorm;
     auto tmon = get_timer("TiogaBlock::update_solution");
 
-    bdata_.qsol_.sync_to_host();
```

Comment:
Instead of deleting this, can you please comment this line and add a `TODO` comment that serves as a reminder to put this back in when we do update TIOGA? It is likely that we will forget about this.

Reply:
@sayerhs Thanks. That is a good suggestion! I have updated the code in https://github.com/sayerhs/tioga_utils/commit/641f61030fc4e146ba9306116cccd9395aa8c893